README.md

```diff
@@ -101,4 +100,4 @@ To check code coverage:
 Licence
 ---
 
-MemDOWN is Copyright (c) 2013-2015 Rod Vagg [@rvagg](https://twitter.com/rvagg) and licensed under the MIT licence. All rights not explicitly granted in the MIT license are reserved. See the included LICENSE file for more details.
+`memdown` is Copyright (c) 2013-2015 Rod Vagg [@rvagg](https://twitter.com/rvagg) and licensed under the MIT licence. All rights not explicitly granted in the MIT license are reserved. See the included LICENSE file for more details.
```

Comment:
Maybe change to 2017? Not sure what to do about this in the future. Sort of a pita to update copyright dates.

Reply:
I'll update. Later, if @rvagg is OK with it, we can just remove the year. In other projects too.